package.json

```diff
@@ -61,7 +61,7 @@
     "test:coverage:jest": "NODE_ENV=test PORT=4243 jest --silent --ci --json --coverage --testLocationInResults --outputFile=\"report.json\" --forceExit --testTimeout=10000",
     "test:updateSnapshot": "NODE_ENV=test PORT=4243 jest --updateSnapshot --testTimeout=10000",
     "seed:setup": "ts-node --compilerOptions '{\"strictNullChecks\": false}' src/test/e2e/seed/segment.seed.ts",
-    "seed:serve": "UNLEASH_DATABASE_NAME=unleash_test UNLEASH_DATABASE_SCHEMA=seed yarn run start:dev",
+    "seed:serve": "DATABASE_NAME=unleash_test DATABASE_SCHEMA=seed yarn run start:dev",
```

Comment:
`yarn seed:serve` also use `start:dev` and it would be better to use the same environment variables instead of using different name such as `UNLEASH_DATABASE_NAME`.